tests/parsers/test_sql_directory_parser.py

```diff
@@ -121,7 +120,9 @@ def test_parse_creates_xcom(self):
                     warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
                 ),
             )
-            aql.render("single_task_dag", input_table=input_table)
+            aql.render(
+                "single_task_dag", input_table=input_table, params={"col": "rooms"}
+            )
```

Comment:
Please add a separate unit test to check that using `{{ params.col }}` produces `room`